shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -250,13 +250,22 @@ private void copy(AlluxioURI srcPath, AlluxioURI dstPath, boolean recursive)
    */
   private void copyFile(AlluxioURI srcPath, AlluxioURI dstPath)
       throws AlluxioException, IOException {
+    FileOutStream os = null;
     try (Closer closer = Closer.create()) {
       OpenFileOptions openFileOptions = OpenFileOptions.defaults().setReadType(ReadType.NO_CACHE);
       FileInStream is = closer.register(mFileSystem.openFile(srcPath, openFileOptions));
       CreateFileOptions createFileOptions = CreateFileOptions.defaults();
-      FileOutStream os = closer.register(mFileSystem.createFile(dstPath, createFileOptions));
+      os = closer.register(mFileSystem.createFile(dstPath, createFileOptions));
       IOUtils.copy(is, os);
       System.out.println("Copied " + srcPath + " to " + dstPath);
+    } catch (Exception e) {
+      if (os != null) {
+        os.cancel();
+        if (mFileSystem.exists(dstPath)) {
+          mFileSystem.delete(dstPath);
```

Comment:
No file will be created if we call `cancel()`, so this shouldn't be necessary

Reply:
Got it, done.